app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :author, class_name: 'User'
+
+  after_save :update_post_counter
+
+  def update_post_counter
+    author.update(posts_counter: author.posts.count)
+  end
```

Comment:
- [ ] **[OPTIONAL]** We can use the [`increment!`](https://apidock.com/rails/ActiveRecord/Base/increment!) method here as well.